website/docs/components/side-panel/examples/portals.tsx

```diff
@@ -10,11 +10,11 @@ const Demo = () => {
     <React.Fragment>
       <Button onClick={() => setVisible(true)}>Show SidePanel</Button>
       <SidePanel visible={visible} onClose={() => setVisible(false)} disablePortal>
-        <SidePanel.Panel mt={20}>
+        <SidePanel.Panel aria-label='Taking the stage' mt={20}>
```

Comment:
Let's remove this mt={20} from the example, because it now looks broken a little bit.

Reply:
decided to keep it, because otherwise this example gets obstructed by the website header